config/metadata/rulesMeta.yaml

```diff
@@ -181,35 +192,35 @@ groups:
         summary: is the value to use when aging out keys.
         description: >
           Indicates the threshold for removing keys from the EMA.
-          The EMA of any key will approach 0 if it is not repeatedly observed,
-          but will never truly reach it, so we have to decide what constitutes
+          The EMA of any key will approach `0` if it is not repeatedly observed,
+          but will never truly reach it, so this field determines what constitutes
           "zero". Keys with averages below this threshold will be removed from
-          the EMA. Default is the same as Weight, as this prevents a key with
+          the EMA. Default is the same as `Weight`, as this prevents a key with
           the smallest integer value (1) from being aged out immediately. This
-          value should generally be <= Weight, unless you have very specific
-          reasons to set it higher.
+          value should generally be less than (<=) `Weight`, unless you 
+          have very specific reasons to set it higher.
       - name: BurstMultiple
         type: float
         summary: is the multiple of the sample rate to use when detecting bursts.
         description: >
           If set, this value is multiplied by the sum of the running average
           of counts to define the burst detection threshold. If total counts
```

Comment:
@kentquirk Is the "burst detection threshold" another named field within this documentation?

Reply:
No, this value defines the threshold used to start burst detection. Basically, if traffic suddenly increases by a factor of X, we want to recalculate the EMA immediately rather than waiting. This number is X.

Reply:
Understood. I asked because my docs senses were on alert given the phrasing around burst detection threshold.